spot-gen/README.txt

```diff
@@ -0,0 +1,5 @@
+Simple data generation
```

Comment:
Can you please add the same usage message as the PR description in this README file?
Also, it'd be great if you make this README.txt as markdown.
Finally, can you please elaborate more on what's the destination of generated files?